arch/arm/core/aarch32/Kconfig

```diff
@@ -215,6 +215,24 @@ config ARM_NONSECURE_FIRMWARE
 	  resources of the Cortex-M MCU, and, therefore, it shall avoid
 	  accessing them.
 
+config ARM_STORE_EXC_RETURN
+	bool
+	default y if FPU_SHARING || ARM_NONSECURE_FIRMWARE
+	help
+	  Store the EXC_RETURN value when switching threads.
+	  This is needed when switching between threads that differ in either
+	  FPU usage or security domain.
+
+config ARM_NONSECURE_PREEMTIBLE_SECURE_CALLS
```

Comment:
@oyvindronningstad by they way I just realized there is a typo here, sorry.
Should  be ARM_NONSECURE_PREEMPTIBLE_SECURE_CALLS